include/circt/Dialect/Moore/MooreOps.td

```diff
@@ -614,6 +614,15 @@ def SExtOp : ExtOpBase<"sext"> {
 // Expressions
 //===----------------------------------------------------------------------===//
 
+def Moore_UArrayCmpPredicateAttr : I64EnumAttr<
```

Comment:
We haven't really followed the pattern of prefixing ops with the dialect as a form of namespacing. We should probably just stick with the current pattern, and change every op in this file separately if we decide to change the convention:
```suggestion
def UArrayCmpPredicateAttr : I64EnumAttr<
```